explorer/api/rpc/[cluster].ts

```diff
@@ -1,7 +1,8 @@
 import type { VercelRequest, VercelResponse } from "@vercel/node";
-import { clusterUrl, Cluster } from "providers/cluster";
+import { clusterUrl, Cluster } from "../../src/providers/cluster";
+import fetch from "node-fetch";
 
-const METHOD_ALLOWLIST = ["getBlock"];
+const METHOD_CACHELIST = ["getBlock"];
```

Comment:
Interesting. My thinking here was that you shouldn't be able to use `/api/rpc/*` except for certain methods. Using `/api/rpc/*` for anything that's not cached just adds a useless layer of indirection between the browser and the RPC. Also it probably costs us money, because Vercel's API layer is probably metered.